src/main.py

```diff
@@ -1,13 +1,16 @@
-from networking import NetworkingManager
+#from networking import NetworkingManager
 from render import RenderingManager
-from database import DatabaseManager
+#from database import DatabaseManager
+from tkinter import *
+from tkinter import ttk
 
 gameState = {"running": True, "stage": "init"} # global stuff like what players on what team, what is the current state of the game (starting, inprogress, finished) and other stuff like that.
 
-databaseManager = DatabaseManager(gameState)
-networkingManager = NetworkingManager(gameState)
-renderingManager = RenderingManager(gameState, networkingManager, databaseManager)
+#databaseManager = DatabaseManager(gameState)
```

Comment:
I would prefer to see these uncommented so they can be used (same with above imports for these classes) before merging to main.